### ISSUE TITLE
[Suggestion/Request] Redragon devices support

### ISSUE THREAD

**Issue by pedrovisck** [antonpup/Aurora#1193]
### What are you experiencing an issue with?

- [ ] Aurora
- [ ] Aurora supported gam
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [ ] Aurora Lighting issue
- [x] Other, (Unsupported device)

### What kind of an issue is it?

- [ ] Crash
- [ ] Performance problems
- [ ] Visual bug
- [ ] Localization issue
- [x] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
It would be interesting if the Redragon keyboards were recognized by Aurora software.

### Actual behavior:
Keyboard not recognized by Aurora. In my case, a k551 Mitra RGB.

### How often does this problem occur?

This problem occurs every time.

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. Keyboard Connected
2. Redragon Manager installed
3. Aurora Installed but not recognizing the keyboard

### What is the version of Aurora you are experiencing this issue with?

My version is the latest

Thanks in advance. 
Best Regards

**Comment by simon-wh** [comment]
Duplicate of #718

**Comment by CalcProgrammer1** [comment]
We finally got a fairly complete build of QMK running on the Redragon keyboards with the OpenRGB HID protocol working.  Here is a short video of my Redragon K552 running QMK, connected to OpenRGB + Aurora running Overwatch.  The lighting works pretty well.

https://www.youtube.com/watch?v=Pf_Q3B1PmK4

I also tested this on the K556.  Most Redragon keyboards use the same Sonix microcontroller, so they should be supportable.  The stock firmware will never be supported for effects like this, as it lacks a direct control mode and is thus useless for real-time effects.  The only solution is to flash a custom replacement firmware.  We've chosen QMK for this as it already exists, is open source, and already runs on a ton of custom keyboards.
